test/unittest/unittest_nntrainer_tensor.cpp

```diff
@@ -1210,6 +1408,87 @@ TEST(nntrainer_Tensor, print_large_size) {
   EXPECT_EQ(ss.str(), expected.str());
 }
 
+TEST(nntrainer_Tensor, DISABLED_private_external_loop_n) {
```

Comment:
any plans to enable this test?

Reply:
I debugged with this test and left it to demonstrate the usage. So, no plans to enable this test. Do you think we should delete this?

Reply:
Enabled.